src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalPrice = 40;
+  const rent = days * rentalPrice;
+  const numberOfDays1 = 3;
+  const numberOfDays2 = 7;
```

Comment:
```suggestion
  const smallDiscountPeriod = 3;
  const bigDiscountPeriod = 7;
```
1 и 2 в названиях ничего не говорят